config/envs/unit_test.py

```diff
@@ -21,3 +21,4 @@ class UnitTestConfig(DefaultConfig):
 
     # notify client passes init with this key and is then mocked
     NOTIFY_API_KEY = "fake_key-0ab1234a-12a3-12ab-12a3-a1b2cd3e4f5g-a123b456-1a23-1abv-a1bc-123a45678910"
+    AUTO_BUILD_ASSETS = True
```

Comment:
What does `AUTO_BUILD_ASSETS` do? Do we want it on for unit tests?

Reply:
@cyrusdobbs `AUTO_BUILD_ASSETS` causes the JS/CSS to be compiled at runtime. Without this we'd need to ensure the build script was run beforehand, else the unit tests will not pass.

Reply:
@cyrusdobbs 
> how come the unit tests depend on the assets?

Because it needs them when we use the test client to render the views.